test/acceptance/Readme.md

```diff
@@ -0,0 +1,59 @@
+# Orion Acceptance Tests
+
+Folder for acceptance tests of Orion.
+
+## How to Run the Acceptance Tests
+
+### Prerequisites:
+
+- Python 2.7
+- pip installed (http://docs.python-guide.org/en/latest/starting/install/linux/)
+- virtualenv installed (pip install virtualenv) (optional).
+Note: We recommend the use of virtualenv, because is an isolated working copy of Python which allows you to work on a specific project without worry of affecting other projects.
+
+##### Environment preparation:
+
+- If you are going to use a virtual environment (optional):
```

Comment:
Is in the "Environment preparation" section, how to create and how to activate it :)